src/annotator/test/sidebar-test.js

```diff
@@ -98,6 +98,7 @@ describe('Sidebar', () => {
     sidebars.forEach(s => s.destroy());
     sandbox.restore();
     $imports.$restore();
+    global.viewport.reset();
```

Comment:
One of the things about magical things in scope (i.e. without importing directly) is that they're magical. I can imagine a future dev happening upon this line and thinking "My goodness, where did `global.viewport` come from?" It may well be a slight ignorance on my part to `karma`'s internals, but I've not seen a pattern of using `global` like this before. Is this a standard karma convention for plugins? I happen to _know_ this comes from `karma-viewport` because of the context of this PR, but I'm not sure how I would have tied the two together without that context. 

Is there any kind of hinting we can give future devs? Is there any kind of karma configuration that would allow us to be more explicit about when we're using plugins in tests (e.g. importing something or flipping some sort of config "switch" on)? 

As a quick aid, perhaps adding comments to the tests where you use this plugin would be immediately helpful.

Reply:
The reason why I use `global.` was because eslint complaints that `viewport` it is not defined. One way of fixing this is to add `karma-viewport` types to `tsconfig.json`, however test files are excluded, so it triggers the error in eslint anyway.